src/app/services/launch-facade.service.ts

```diff
@@ -15,19 +16,36 @@ export class LaunchFacadeService {
   launchListLoaded$ = this.store.select(launchListQuery.getLaunchListLoaded);
   launchListLoading$ = this.store.select(launchListQuery.getLaunchListLoading);
 
+  launchDetailState$ = this.store.select(launchDetailQuery.getLaunchDetailState);
+  launchDetail$ = this.store.select(launchDetailQuery.getLaunchDetail);
+  launchDetailLoaded$ = this.store.select(launchDetailQuery.getLaunchDetailLoaded);
+  launchDetailLoading$ = this.store.select(launchDetailQuery.getLaunchDetailLoading);
+
   constructor(
     private readonly store: Store<LaunchListState>,
-    private readonly pastLaunchesService: PastLaunchesListGQL
+    private readonly pastLaunchesService: PastLaunchesListGQL,
+    private readonly launchesDetailService: LaunchDetailsGQL
   ) {}
 
   pastLaunchListStoreCache() {
     this.store.dispatch(loadLaunchList());
     return this.launchList$;
   }
 
+  pastLaunchDetailsStoreCache(data) {
+    this.store.dispatch(loadLaunchDetail(data));
+    return this.launchDetail$;
+  }
+
   pastLaunchListFacade() {
     return this.pastLaunchesService
       .fetch({ limit: 30 })
       .pipe(map(res => res.data.launchesPast));
   }
+
+  pastLaunchDetailFacade(id) {
+    return this.launchesDetailService
+      .fetch({id: id})
```

Comment:
Instead if `id: id` we can also write as `id`

Reply:
Fixed